### PR TITLE
Doc review

### PR DESCRIPTION
**Pull request by jumeyermattosarruda**
#### What is the purpose of this pull request?
I'm adding design guidelines as documented in this [Notion](https://www.notion.so/vtexhandbook/9469d4e73b314df7907d6ee3fe7d005d?v=77e2e8a871484d6c8ca13c1374649a8e) page. I added a new folder for design guidelines. The PR will help me track the preview in Admin UI site, and check with designers the proposed changes before merging it. 

#### What problem is this solving?
We need to include design recommendations for components, so designers can make better, informed decisions when using our design system.

#### How should this be manually tested?
Check preview of docs.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [x] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: packages/admin-ui-docs/docs/design guidelines/form.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## Best practices and general recommendations:
+
+- Sections should have at least three fields.
+- Use section description only when necessary and the additional content is essential.
+- If there’s only one section in a page, no section header should exist.
+
+## MultiSelect (to be implemented)
+
+- Use when multiple options need to be selected from a list larger than 7, or from a dynamic list of any size.
+- Example
```
Comment (kevinch) [inline packages/admin-ui-docs/docs/design guidelines/form.md:32]: Missing something here?

Comment (jumeyermattosarruda) [reply packages/admin-ui-docs/docs/design guidelines/form.md]: Fixed!

**Comment by jumeyermattosarruda** [comment]
Rolling back the previous structure for design guidelines, after a chat with Davis, since they will be added in the context of each component, under the design tab. Will move on to reviewing each component specifically instead of adding general design guidelines for now. 

Also added the "Contributing to docs" section on the `/onboarding` page, as suggested by @lucasaarcoverde.

File: packages/admin-ui-docs/docs/design guidelines/form.md (hunk 1 above)
Comment (jumeyermattosarruda) [reply packages/admin-ui-docs/docs/design guidelines/form.md]: Corrected!

**Comment by jumeyermattosarruda** [comment]
Hey @kevinch! Can you please review this PR?

File: packages/admin-ui-docs/docs/teams/contributing/onboarding.mdx [hunk 2]
```diff
@@ -51,19 +54,55 @@ Follow the instructions below to get started on development.
...
 6.  Format your code with prettier (yarn prettier).
 7.  Make sure your code lints (yarn lint).
 8.  Run `yarn build`, yarn `build:storybook` or `yarn build:site` to ensure that the builds are still working.
+
+
+## Contributing with documentation
+Our Admin UI documentation is constantly updated and improved to support you in your journey with the VTEX design system. And to make this a collaborative effort, check out how to contribute with our documentation in the sections below.
+
+### Editing a page
+Our documentation site is built using [Docusaurus 2](https://docusaurus.io/), a static website generator. All of our documentation is stored in the [/docs folder](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-docs/docs) of our Github Repository.
```
Comment (kevinch) [inline packages/admin-ui-docs/docs/teams/contributing/onboarding.mdx:79]:
```suggestion
Our documentation site is built using [Docusaurus](https://docusaurus.io/), a static website generator. All of our documentation is stored in the [/docs folder](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-docs/docs) of our Github Repository.
```

I'd remove the version. what matters here is the tool, not its version. Plus if we update in the future to the 3x version, we're covered.

File: packages/admin-ui-docs/docs/teams/developing/code-styleguide.mdx [hunk 3]
```diff
@@ -280,7 +280,7 @@ Responsive aliases are preferred over custom media queries and responsive arrays
 
 ### Avoid parent styles
 
-While coding an app it doesn't matter, but if you are within a library this is a bad practice! The user will not be able to customize the child's style without the `!important` flag. So, you must avoid it.
+Avoiding parent styles While coding an app does not matter much, but if you are within a library this is a bad practice. The user will not be able to customize the child's style without the `!important` flag. So, you must avoid it.
```
Comment (kevinch) [inline packages/admin-ui-docs/docs/teams/developing/code-styleguide.mdx:283]:
```suggestion
Avoiding parent styles while coding an app does not matter much, but if you are within a library this is a bad practice. The user will not be able to customize the child's style without the `!important` flag. So, you must avoid it.
```

**Review by kevinch — APPROVED** [review]

**Pull request merged**